community/kernel/src/main/java/org/neo4j/kernel/impl/transaction/tracing/TransactionEvent.java

```diff
@@ -81,7 +81,7 @@ public void setReadOnly( boolean wasReadOnly )
 
     /**
      * Set the type of the transaction, as given by
-     * {@link org.neo4j.kernel.impl.api.KernelTransactionImplementation.TransactionType}.
+     * {@link org.neo4j.kernel.impl.api.KernelTransactionImplementation.TransactionWriteState}.
      */
     void setTransactionType( String transactionTypeName );
```

Comment:
`TransactionEvent` has no implementations that is not `no_op`. Is this part of public API? If so, why does it live in `impl`? If not, can we just delete it completely?